Ecosystem Partners/getting-started-with-vormetric-dsm.md

```diff
@@ -53,10 +53,15 @@ This deployment process for Partner Templates currently requires manual interact
 ### Postrequisite
 - If you want to access your Vormetric DSM over the internet, please perform the following tasks once your Vormetric DSM has been deployed to your account:
 1. Adding an external ip-address to an existing or new CenturyLink server
-![ip.jpg](https://t3n.zendesk.com/attachments/token/kObGC9P2IjP1ate0NexwFNiXz/?name=ip.jpg)
```

Comment:
I never knew what that tag was for.  Can you please let it slide this one time and I will make the updates?  I probably have several more to make and would like to swoop through to do them all at once this week.

Thanks again for the tip!!  Please keep them coming.

Sincerely,

Bob Stolzberg | Sr. Cloud Integration Engineer
CenturyLink
Mobile/Text: (314) 397-2610
E-Mail: bob@CTL.io
www.CenturylinkCloud.com


From: Nathan Young <notifications@github.com<mailto:notifications@github.com>>
Reply-To: CenturyLinkCloud/PublicKB <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wed, 11 Mar 2015 14:33:16 -0700
To: CenturyLinkCloud/PublicKB <PublicKB@noreply.github.com<mailto:PublicKB@noreply.github.com>>
Cc: User <bob.stolzberg@centurylink.com<mailto:bob.stolzberg@centurylink.com>>
Subject: Re: [PublicKB] Bob stolzberg test (#34)


In Ecosystem Partners/getting-started-with-vormetric-dsm.md<https://github.com/CenturyLinkCloud/PublicKB/pull/34#discussion_r26258056>:

> @@ -53,10 +53,15 @@ This deployment process for Partner Templates currently requires manual interact
>  ### Postrequisite
>  - If you want to access your Vormetric DSM over the internet, please perform the following tasks once your Vormetric DSM has been deployed to your account:
>  1. Adding an external ip-address to an existing or new CenturyLink server
> -![ip.jpg](https://t3n.zendesk.com/attachments/token/kObGC9P2IjP1ate0NexwFNiXz/?name=ip.jpg)


The text in between the brackets is for img alt. The purpose of alt text is for screen readers to describe what the image is of. To a blind person, having a screen reader say "ip.jpg" isn't all that useful ;). Can you fix these so that it doesn't just display the image file name?

-
Reply to this email directly or view it on GitHub<https://github.com/CenturyLinkCloud/PublicKB/pull/34/files#r26258056>.

This communication is the property of CenturyLink and may contain confidential or privileged information. Unauthorized use of this communication is strictly prohibited and may be unlawful. If you have received this communication in error, please immediately notify the sender by reply e-mail and destroy all copies of the communication and any attachments.